src/amchart/Area.js

```diff
@@ -6,35 +11,73 @@
         root.amchart_Area = factory(root.d3, root.amchart_CommonSerial, root.amcharts, root.api_INDChart);
     }
 }(this, function(d3, CommonSerial, AmCharts, INDChart) {
+    /**
+     * @class amchart_Area
+     * @extends amchart_CommonSerial
+     * @extends api_INDChart
```

Comment:
Can we fix the template as its not just a matter of "showing up" it needs to be correct when it does show up!